src/lib.rs

```diff
@@ -120,47 +174,31 @@ pub fn build_from_docker(
         signing_certificate,
         private_key,
     )
-    .map_err(|err| {
-        new_nitro_cli_failure!(
-            &format!("Failed to create EIF image: {:?}", err),
-            NitroCliErrorEnum::EifBuildingError
-        )
-    })?;
+    .map_err(|err| new_nitro_cli_failure!(
+        &format!("Failed to create EIF image: {:?}", err),
+        NitroCliErrorEnum::EifBuildingError
+    ))?;
 
     if let Some(docker_dir) = docker_dir {
+        info!("Building docker image, this will take a moment");
         docker2eif
             .build_docker_image(docker_dir.clone())
-            .map_err(|err| {
-                new_nitro_cli_failure!(
-                    &format!("Failed to build docker image: {:?}", err),
-                    NitroCliErrorEnum::DockerImageBuildError
-                )
-            })?;
+            .map_err(|err| new_nitro_cli_failure!(
+                &format!("Failed to build docker image: {:?}", err),
+                NitroCliErrorEnum::DockerImageBuildError
+            ))?;
     } else {
-        docker2eif.pull_docker_image().map_err(|err| {
-            new_nitro_cli_failure!(
-                &format!("Failed to pull docker image: {:?}", err),
-                NitroCliErrorEnum::DockerImagePullError
-            )
-        })?;
+        info!("Pulling docker image `{}`", docker_uri);
+        docker2eif.pull_docker_image().map_err(|err| new_nitro_cli_failure!(
+            &format!("Failed to pull docker image: {:?}", err),
+            NitroCliErrorEnum::DockerImagePullError
+        ))?;
     }
-    let measurements = docker2eif.create().map_err(|err| {
-        new_nitro_cli_failure!(
-            &format!("Failed to create EIF image: {:?}", err),
-            NitroCliErrorEnum::EifBuildingError
-        )
-    })?;
-    eprintln!("Enclave Image successfully created.");
-
-    let info = EnclaveBuildInfo::new(measurements.clone());
-    println!(
-        "{}",
-        serde_json::to_string_pretty(&info).map_err(|err| new_nitro_cli_failure!(
-            &format!("Failed to display EnclaveBuild data: {:?}", err),
-            NitroCliErrorEnum::SerdeError
-        ))?
-    );
```

Comment:
The `EnclaveBuildInfo` output seems to have been removed. We should probably keep it or move it to `main.rs`, if we want to upstream these changes.

Reply:
Moved it to `main.rs`